### PR TITLE
Added explicit padding to ensure consistency with PyTorch. Addresses #4

### PR DESCRIPTION
**Pull request by ujjwalongithub**
TensorFlow SAME padding is not the same as the explicit padding in PyTorch.

In the SAME padding setup of TensorFlow, the values may be unequally padded on different sides, which is not the case with PyTorch.

In this pull request we add explicit padding to ensure consistency with the original implementation here : 

https://github.com/kenshohara/3D-ResNets-PyTorch/blob/master/models/resnet.py

### CODE REVIEW

**Comment by ujjwalongithub** [comment]
This addresses #4  and aims to correct the previous merge request of #3

**Review by ujjwalongithub — COMMENTED** [review]
This seems fine and should be merged.
